### DISCUSSION TITLE
有没有考虑增加Claude的代理

### DISCUSSION THREAD

**Discussion by honeyshine75** [BurnHair/docs#197] [功能建议]
很想体验下Claude，但封号很严重

**Comment by BennyThink** [comment]
暂时还没，以后也许会🤔

**Reply by q741097322** [reply]
佬，能提上日程吗，最近其他的claude3.5都用不了了（哭

**Reply by narugakuru** [reply]
今天用了一下1022版sonnet，只是简单提了一下功能函数要求，它自动把函数功能划分，日志和异常处理，参数列表说明全都写好了，用着非常舒服，但封号太严重了，佬啥时候增加claude的api啊（急急急

**Comment by zakizou** [comment]
期待claude3.5

**Comment by fevrax** [comment]
期待claude3.5， 比 gpt 好用多了

**Comment by BennyThink** [comment]
https://github.com/orgs/BurnHair/discussions/207

好了
